src/components/base/outline-card/outline-card.stories.ts

```diff
@@ -96,7 +96,7 @@ const Template = ({
   html`
     <outline-container>
       <outline-card
-        hasshadow="${ifDefined(hasShadow)}"
+        ?hasshadow=${ifDefined(hasShadow)}
```

Comment:
Thanks for this. In reviewing the PR I double checked and realized this worked for the alert code. :)

Reply:
Oh nice, yeah I wanted to talk about that during the dev huddle today. I cleared up some confusion I had around the data binding.